CMSgov/design-system
# [WNMGDS-1148] stories for FilterChip, Drawer & FormLabel

## Summary
Added storybook stories for FilterChip, Drawer & FormLabel

## How to test
Run `yarn storybook` and see newly added stories. I tried to encapsulate some of the major examples while relying on controls for more variations
